### PR TITLE
Fix of Cypress Tests Due To NavBar & Include Ext. Data Changes

### PR DESCRIPTION
**Pull request by ugur-camoglu**

### CODE REVIEW

File: deploy/post_deploy_testing/cypress/integration/03a_browse_views_basic.js [hunk 1]
```diff
@@ -5,40 +5,42 @@ describe('Browse Views - Basic Tests', function () {
...
                 cy.get('#page-title-container .page-title span.title').should('have.text', 'Data Browser');
             });
 
         });
 
-        it('Only shows award.project=4DN results & "Include External Data" is off', function(){
-
-            cy.location('search').should('include', 'award.project=4DN');
-            cy.get('#stats .browse-base-state-toggle-container input[type="checkbox"]').should('not.be.checked');
+        it('Only shows award.project=4DN results & "Include External Data" is off', function(){
```
Comment (j1z0) [inline deploy/post_deploy_testing/cypress/integration/03a_browse_views_basic.js]: Codacy Issue found: [Trailing spaces not allowed. (no-trailing-spaces)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4521913)

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Clones removed
==============
+ deploy/post_deploy_testing/cypress/integration/03b_browse_views_barplot_as_filter_control.js  -1
+ deploy/post_deploy_testing/cypress/integration/03c_browse_views_barplot_aggregations.js  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/fourfront/fourfront/pullRequest?prid=4521913&bid=15229375)

File: deploy/post_deploy_testing/cypress/integration/03a_browse_views_basic.js [hunk 2]
```diff
@@ -5,40 +5,42 @@ describe('Browse Views - Basic Tests', function () {
     context('Navigation and Redirection', function(){
 
         it('If start from home page, clicking on Browse nav menu item gets us to Browse page.', function(){
-
+            const dataNavBarItemSelectorStr = '#top-nav div.navbar-collapse .navbar-nav a.id-data-menu-item';
             cy.visit('/');
+            cy.get(dataNavBarItemSelectorStr).click().wait(500).then(()=>{
+
```
Comment (j1z0) [inline deploy/post_deploy_testing/cypress/integration/03a_browse_views_basic.js]: Codacy Issue found: [Trailing spaces not allowed. (no-trailing-spaces)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4521913)

File: deploy/post_deploy_testing/cypress/integration/03a_browse_views_basic.js [hunk 3]
```diff
@@ -5,40 +5,42 @@ describe('Browse Views - Basic Tests', function () {
     context('Navigation and Redirection', function(){
 
         it('If start from home page, clicking on Browse nav menu item gets us to Browse page.', function(){
-
+            const dataNavBarItemSelectorStr = '#top-nav div.navbar-collapse .navbar-nav a.id-data-menu-item';
             cy.visit('/');
+            cy.get(dataNavBarItemSelectorStr).click().wait(500).then(()=>{
```
Comment (alexkb0009) [inline deploy/post_deploy_testing/cypress/integration/03a_browse_views_basic.js]: Was this unfinished / forgot to push recent edits?
Looks like added new selector but not using it?

Was also gonna say maybe could put `dataNavBarItemSelectorStr` into `'./../support/variables';` so would be imported in same way as `navBrowseBtnSelector` was but this doesn't really matter unless need to use `dataNavBarItemSelectorStr` in some other files (which is possible / don't remember).

Comment (utku-ozturk) [reply deploy/post_deploy_testing/cypress/integration/03a_browse_views_basic.js]: `dataNavBarItemSelectorStr` selector is used only once, assigned as const is redundant. Removed.

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Clones removed
==============
+ deploy/post_deploy_testing/cypress/integration/03c_browse_views_barplot_aggregations.js  -1
+ deploy/post_deploy_testing/cypress/integration/03b_browse_views_barplot_as_filter_control.js  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/fourfront/fourfront/pullRequest?prid=4521913&bid=15229375)

**Pull request merged**